### PR TITLE
Fixed configurations to use data and config file as volume

### PR DESCRIPTION
**Pull request by CometKim**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,9 +19,12 @@ WORKDIR /mm
 # Copy over files
 ADD https://releases.mattermost.com/3.4.0-rc3/mattermost-team-3.4.0-rc3-linux-amd64.tar.gz .
 RUN tar -zxvf ./mattermost-team-3.4.0-rc3-linux-amd64.tar.gz
-ADD config_docker.json ./mattermost/config/config_docker.json
-ADD docker-entry.sh .
```
Comment (CometKim) [inline Dockerfile]: review

File: Dockerfile [hunk 2]
```diff
@@ -19,9 +19,12 @@ WORKDIR /mm
 # Copy over files
 ADD https://releases.mattermost.com/3.4.0-rc3/mattermost-team-3.4.0-rc3-linux-amd64.tar.gz .
 RUN tar -zxvf ./mattermost-team-3.4.0-rc3-linux-amd64.tar.gz
-ADD config_docker.json ./mattermost/config/config_docker.json
-ADD docker-entry.sh . 
 
+RUN mkdir ./mattermost-config
+VOLUME ./mattermost-config
```
Comment (CometKim) [inline Dockerfile]: comment

**Comment by jasonblais** [comment]
Thanks @CometKim ! 

Could we have your help join the [approved contributor's list](http://www.mattermost.org/mattermost-contributor-agreement/) so this PR can be reviewed and merged? 

Also, please help update the PR to reflect the latest updates on `master`.

Once again, really sorry for the long delay in the review process.

**Comment by CometKim** [comment]
please move to #15

**Pull request closed without merging**